### PR TITLE
add colors for admin workflows page

### PR DESCRIPTION
**Pull request by taofeeq-deru**
This PR updates tailwind config colors and adds colors to workflows pages

### CODE REVIEW

File: packages/admin/tailwind.config.ts [hunk 1]
```diff
@@ -14,6 +14,57 @@ const config = {
...
+          'border-destructive': 'hsl(3deg, 72.4%, 51.6%)', //colors should be hsl/oklch values
+          'el-1': '#5c5c5f',
+          'el-2': '#707070',
+          'el-3': '#939393',
+          'el-4': '#a9a9a9',
+          'el-5': '#e6e6e6',
+          'el-6': '#ffffff',
+          'el-accent': '#5699a8',
+
+          stages: {
```
Comment (adeleke5140) [inline packages/admin/tailwind.config.ts]: we can take out tags and stages for now

Comment (taofeeq-deru) [reply packages/admin/tailwind.config.ts]: okay
on it

Comment (taofeeq-deru) [reply packages/admin/tailwind.config.ts]: done

**Review by adeleke5140 — APPROVED** [review]
lgtm

**Pull request merged**